cmd/proxy-server/main.go

```diff
@@ -540,3 +563,152 @@ func rolesHandler(log *logrus.Entry, opaHost string) http.Handler {
 		defer res.Body.Close()
 	})
 }
+
+func volumesHandler(roleServ *roleClientService, tm token.Manager, log *logrus.Entry) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Can a log statement be added to indicate that this specific handler is being served for a tenant? In the handlers for the drivers, we have debug logs like this:

` level=debug msg="Serving request" endpoint="https://10.0.0.1" system_id=6daebab67338ed0f`

We can have a log message like "Serving volumes request for tenant <tenant> for storage type <storage type>".